src/components/ProductsListContainer.js

```diff
@@ -0,0 +1,40 @@
+import productsData from '../mocks/en-us/featured-products.json';
+import Product from '../components/Product.js';
+import Paginator from './Paginator.js';
+
+const ProductsListContainer = (props) => {
+    const paginator = (props.hasPaginator !== undefined) ? <Paginator /> : <span/>;
```

Comment:
If you change it to 
```suggestion
    const paginator = (props.hasPaginator) ? <Paginator /> : <span/>;
```
also works and remember what i wrote before in other comments about props of components